sspl_test/plans/self_primary.pln

```diff
@@ -1,4 +1,4 @@
 alerts.self_hw.test_self_hw_real_stor_enclosure_conn
 alerts.node.test_node_fan_actuator
 alerts.self_hw.test_self_hw_node_sel_event
-alerts.realstor.test_real_stor_fan_actuator
\ No newline at end of file
+alerts.realstor.test_real_stor_fan_actuator
```

Comment:
no change here, file should not reflect in PR

Reply:
Done.